docs/src/functions.md

```diff
@@ -75,3 +75,9 @@ Maximum
 Quadratic
 SumPositive
 ```
+
+## Distances
```

Comment:
This I have also put in the section describing calculus rules, since these are really modifiers of some other (indicator) functions; but I guess they can stay in both places.

Reply:
They were in the FUNCTIONS.md in the root, so I copied them here. But you are right, we should think of how we can unify the different documentations.